src/compile.coffee

```diff
@@ -78,7 +77,7 @@ findRelevantTransactions = (mediaType, refract, apiElements) ->
       # of Dredd. There's a plan to migrate to so-called "transaction paths"
       # in the future (apiaryio/dredd#227), which won't use the concept
       # of transaction examples anymore.
-      exampleNumbersPerTransaction = detectExampleNumbersPerTransaction(transitionElement)
+      exampleNumbersPerTransaction = detectTransactionExampleNumbers(transitionElement)
```

Comment:
I wonder which name describes the function better - whether the result is "example numbers per transaction" or "transaction example numbers". I'm not quite sure myself.